scripts/submit-analytics.js

```diff
@@ -259,7 +239,25 @@ async function submitAnalytics(dryRun) {
     return;
   }
 
-  await Promise.all([dispatchAnalytics(payload)]);
+  return new Promise((resolve, reject) => {
+    // node 19 turns on keep-alive by default and it will make the https.get() to hang
+    // https://github.com/realm/realm-js/issues/5136
+    https.globalAgent = new https.Agent({ keepAlive: false });
```

Comment:
If we move the `https` import to this function, then it may be safer to override the `globalAgent`. E.g. thinking of a scenario where `https` is global in this file and by passing `{ keepAlive: false }` would set the other non-specified `option` values to their defaults right? (And perhaps override any changes made to these somewhere else in this file). Although, I assume this is an unlikely scenario. But localizing the `https` might still be favorable.

Reply:
I have accepted the suggestion by @nirinchev which is basically doing what you are saying.